tests/unit/amo/pages/TestBlock.js

```diff
@@ -213,7 +213,22 @@ describe(__filename, () => {
     expect(root.find('.Block-reason')).toHaveLength(0);
   });
 
-  it('renders the min/max versions', () => {
+  it('renders the min/max versions if min is 0 and max is *', () => {
```

Comment:
it `renders "all versions" when min is 0 and max is *`

Reply:
:+1: